site/ChartsIndexPage.tsx

```diff
@@ -152,6 +152,7 @@ export const ChartsIndexPage = (props: {
                 </main>
                 <SiteFooter baseUrl={baseUrl} />
                 <script
+                    type="module"
```

Comment:
I've added `type="module"` to all our inline scripts, reason being:

We now load our assets as `script type="module"`, and in my experiments the only way to guarantee that the inline script is executed after all the others are loaded is to also make the inline script `type="module"`.